docs/src/main/asciidoc/datasource.adoc

```diff
@@ -511,154 +478,247 @@ public class TestResources {
 }
 ----
 
-This will allow you to test your application even when it's compiled into a native executable,
-while the database will run in the JVM as usual.
-
-Connect to it using:
-
+. Connect to it using:
++
 [source,properties]
 ----
 quarkus.datasource.db-kind=h2
-quarkus.datasource.jdbc.url=jdbc:h2:mem:test
+quarkus.datasource.jdbc.url=jdbc:h2:tcp://localhost/mem:test
 ----
 
+[[datasource-reference]]
+== References
+
 [[configuration-reference]]
-== Common Datasource Configuration Reference
+=== Common datasource configuration reference
 
 include::{generated-dir}/config/quarkus-datasource.adoc[opts=optional, leveloffset=+1]
 
 [[jdbc-configuration]]
-== JDBC Configuration Reference
+=== JDBC configuration reference
 
 include::{generated-dir}/config/quarkus-agroal.adoc[opts=optional, leveloffset=+1]
 
 [[jdbc-url]]
-== JDBC URL Reference
+=== JDBC URL reference
```

Comment:
Possibly for a later iteration, could the content in the  ==== headings be restructured into a table for comparison?